src/languages/es.js

```diff
@@ -36,6 +36,10 @@ export default {
         contacts: 'Contactos',
         recents: 'Recientes',
         close: 'Cerrar',
+        download: 'Descargar',
+        pin: 'Anclar',
```

Comment:
I would suggest using `Fijar`. This is the terminology used in popular apps like WhatsApp and Twitter.